test/functional/ui/wildmode_spec.lua

```diff
@@ -52,6 +54,59 @@ describe("'wildmode'", function()
         :!^                       |
       ]])
     end)
+
+    it('is preserved during :terminal activity', function()
+      command('set wildmenu wildmode=full')
+      command('set scrollback=4')
+      if iswin() then
+        if helpers.pending_win32(pending) then return end
+        -- set_shell_powershell()
+        -- feed([[:terminal 1,2,3,4,5 | foreach-object -process {echo $_; sleep 0.1}]])
+      else
+        feed([[:terminal for i in $(seq 1 5000); do printf 'foo\nfoo\nfoo\n'; sleep 0.1; done<cr>]])
+      end
+      screen:sleep(50)        -- Allow some output.
+      feed([[<C-\><C-N>gg]])
+      feed([[:sign <Tab>]])   -- Invoke wildmenu.
+      screen:sleep(50)        -- Allow some output.
```

Comment:
The `screen:sleep()` here and in the next test are necessary to make sure some activity was allowed. But it also causes `screen:expect()` to wait until the full timeout. @bfredl I applied https://github.com/neovim/neovim/pull/6930 and found that it fixed that issue for both of these cases.